### PR TITLE
[IOTDB-5028] Using `show timeseries` to query the schema in import-csv

### PR DESCRIPTION
**Pull request by xuanronaldo**

### CODE REVIEW

File: cli/pom.xml [hunk 1]
```diff
@@ -79,6 +79,11 @@
             <artifactId>iotdb-antlr</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.iotdb</groupId>
+            <artifactId>node-commons</artifactId>
+            <version>0.14.0-SNAPSHOT</version>
+        </dependency>
```
Comment (HTHou) [inline cli/pom.xml:86]: Already have this dependency in L108

**Review by HTHou — CHANGES_REQUESTED** [review]

File: cli/pom.xml (hunk 1 above)
Comment (xuanronaldo) [reply cli/pom.xml:86]: Ok, I got it.

File: cli/src/main/java/org/apache/iotdb/tool/ImportCsv.java [hunk 2]
```diff
@@ -817,32 +826,37 @@ private static void parseHeaders(
...
-          String[] split = columnNames.get(i).split("\\.");
-          String measurement = split[split.length - 1];
-          headerTypeMap.put(measurement, getType(columnTypes.get(i)));
+    boolean hasResult = false;
+    for (String deviceName : deviceNames) {
+      String sql = "show timeseries " + deviceName + ".*";
+      SessionDataSet sessionDataSet = null;
+      try {
+        sessionDataSet = session.executeQueryStatement(sql);
+        int tsIndex = sessionDataSet.getColumnNames().indexOf("Timeseries");
```
Comment (HTHou) [inline cli/src/main/java/org/apache/iotdb/tool/ImportCsv.java:837]:
```suggestion
        int tsIndex = sessionDataSet.getColumnNames().indexOf(ColumnHeaderConstant.TIMESERIES);
```

File: cli/src/main/java/org/apache/iotdb/tool/ImportCsv.java [hunk 3]
```diff
@@ -817,32 +826,37 @@ private static void parseHeaders(
...
+        int dtIndex = sessionDataSet.getColumnNames().indexOf("DataType");
```
Comment (HTHou) [inline cli/src/main/java/org/apache/iotdb/tool/ImportCsv.java:838]:
```suggestion
        int dtIndex = sessionDataSet.getColumnNames().indexOf(ColumnHeaderConstant.DATATYPE);
```

File: cli/src/main/java/org/apache/iotdb/tool/ImportCsv.java (hunk 2 above)
Comment (xuanronaldo) [reply cli/src/main/java/org/apache/iotdb/tool/ImportCsv.java]: changed

File: cli/src/main/java/org/apache/iotdb/tool/ImportCsv.java (hunk 3 above)
Comment (xuanronaldo) [reply cli/src/main/java/org/apache/iotdb/tool/ImportCsv.java]: changed

**Review by HTHou — APPROVED** [review]

**Pull request merged**